index.js

```diff
@@ -221,11 +242,20 @@ class MarkdownToolbarElement extends HTMLElement {
   }
 
   connectedCallback() {
+    if (!this.hasAttribute('role')) {
+      this.setAttribute('role', 'toolbar')
+    }
+    this.addEventListener('keydown', focusKeydown)
     const fn = shortcut.bind(null, this)
     if (this.field) {
       this.field.addEventListener('keydown', fn)
       shortcutListeners.set(this, fn)
     }
+    let tabindex = '0'
+    for (const button of getButtons(this)) {
+      button.setAttribute('tabindex', tabindex)
+      if (tabindex === '0') tabindex = '-1'
+    }
```

Comment:
This block wouldn't work for toolbars that were connected but were hidden in its default state, like the toolbars in our comment edit forms. `getButtons()` would be empty.

Perhaps we keep setting tabindex on md buttons in connected callback, and run this block once on a focus event?

Reply:
Rather than setting tabindex on md buttons, I've made the toolbar itself initially focussable, and upon focus it moves focus to the nearest button.